packages/v4/src/content/design-guidelines/usage-and-behavior/card-view/cards.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Cards
+section: usage and behavior
+hideTOC: true
+showTitle: true
+---
+
+A **card** is a square or rectangular container of related information. Cards symbolize units of information, and each one acts as an entry point for users to access more details. For example, in dashboards and catalog views, cards function as a preview of a detailed page.  
+
+A card usually consists of  3 parts: 
+
+<img src="./img/card.png"  alt="3 parts of a card"  width="394"/>  
+
+
+1. **Header**: Title of the information group
+2. **Body**: Brief details about the group
+3. **Footer**: Links to access more information about the group 
+
+Cards are flexible, and you can customize them in a variety of ways:
+- A card can contain a header with an image, actions, and a body.
+- A card can contain only a header and a body. 
+- A card can contain a header with an image, a body, and footer. 
+
+## Card types 
+PatternFly supports two types of cards: 
+1. **Compact (recommended)**: Compact cards use 16px spacing with an 8px spacer between the header and body and 16px between the body and footer.
+2. **Large**: Large cards use 24px spacing with a 16px spacer between the header and body and 24px between the body and footer.  
+
+<img src="./img/card-types.png"  alt="compact and standard card types"  width="779"/> 
+
+## Usage 
+Use a card:
+- To visually group content or distinguish information from the rest of a page’s content. Examples include dashboards and catalog views. 
+- To organize content in layouts with multiple columns or grids.  
+- To compare separate groups of information.  
+- To make media such as images and videos easier to scan.  
+
+<img src="./img/dashboard.png"  alt="card view with charts"  width="990"/> 
+
+Do NOT use a card:
+- Behind a component, like a data table or a list, when it's the only section on the page. (But when there are both cards and tables on the page, you can separate the page using cards.) 
```

Comment:
Can we strike the first 2 bullets here and the screen shot below? Due to some implementation issues, we are rethinking changes to card usage for full screen views.

Reply:
Thanks, @mcarrano ! I removed those bullet items and the screenshot.